### PR TITLE
Update Category Controller And Added Constructor

### PR DESCRIPTION
**Pull request by uzzal71**
Code Now Pull

### CODE REVIEW

File: app/Http/Controllers/CategoryController.php [hunk 1]
```diff
@@ -6,6 +6,19 @@
 
 class CategoryController extends Controller
 {
+        /**
+     * Display a listing of the resource.
```
Comment (uzzal71) [inline app/Http/Controllers/CategoryController.php:10]: Change this comment lines 01

File: app/Http/Controllers/CategoryController.php [hunk 2]
```diff
@@ -6,6 +6,19 @@
...
+     *
+     * @return \Illuminate\Http\Response
```
Comment (uzzal71) [inline app/Http/Controllers/CategoryController.php:12]: Change Comment 02 Not a Response

**Review by uzzal71 — COMMENTED** [review]
Change Two File

File: app/Http/Controllers/CategoryController.php (hunk 1 above)
Comment (uzzal649) [reply app/Http/Controllers/CategoryController.php:10]: Done

File: app/Http/Controllers/CategoryController.php (hunk 2 above)
Comment (uzzal649) [reply app/Http/Controllers/CategoryController.php:12]: Done

**Review by uzzal649 — COMMENTED** [review]
All Changed Done 06/15/2022

File: app/Http/Controllers/CategoryController.php (hunk 2 above)
Comment (uzzal71) [reply app/Http/Controllers/CategoryController.php:12]: ok i will check

File: app/Http/Controllers/CategoryController.php (hunk 1 above)
Comment (uzzal71) [reply app/Http/Controllers/CategoryController.php:10]: I Will Check

**Pull request merged**